docs/concepts/privacy.md

```diff
@@ -0,0 +1,71 @@
+
+This is one of the advantages of IPFS over traditional legacy-web hosting. It means retrieving files — especially popular ones that exist on lots of nodes in the network — can be faster and more bandwidth-efficient. However, it's important to note that those DHT queries happen in public. Because of this, it's possible that third parties could be monitoring this traffic to determine what CIDs are being requested, when, and by whom. As IPFS continues to grow in popularity, it's more likely that such monitoring will exist.
+
+### Node IDs and node requests
+
+The other half of the equation when considering the prospect of IPFS traffic monitoring is that nodes' unique identifiers are themselves public. Just like with CIDs, every individual IPFS node has its own public identifier (known as a PeerID), such as `QmRGgYP1P5bjgapLaShMVhGMSwGN9SfYG3CM2TfhpJ3igE`.
+
+While a long string of letters and numbers may not be a "Johnny Appleseed" level of human-readable specificity, your PeerID is still a long-lived, unique identifier for your node. Keep in mind that it's possible to do a DHT lookup on your PeerID and, particularly if your node is regularly running from the same location (like your home), find your IP address. Additionally, longer-term monitoring of the public IPFS network could yield information about what CIDs your node is requesting and/or reproviding and when.
+
+## Enhancing your IPFS privacy
+
+If there are situations in which you know you'll need to remain private but still want to use IPFS, one of the approaches outlined below may help. And don't forget, you can always discuss privacy and get others' input or ideas in the official [IPFS forums](https://discuss.ipfs.io).
+
+### Disabling reproviding
+
+By default, an IPFS node announces itself as a provider for every CID in its cache. If you'd like to disable this behavior, you can do so in the [reprovider settings](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#reprovider) of your node's config file.
+
+Disabling default reprovision still enables you to provide data that you've pinned, so it's still possible for you to provide other nodes with content that you care about and want to make sure continues to be available over IPFS.
```

Comment:
reproviding?

The sentence is confusing after reading the above section. Kinda sounds like "You just turned off the sharing but it still shares". Need some disambiguation between automatic providing vs ... whatever the alternate ways to share are. Explicit pins maybe?

Reply:
https://github.com/ipfs/ipfs-docs/pull/554/commits/d0f0e3e3149fcea46a3b9931f0e0b7e27d48595e should clarify this.